### ISSUE TITLE
[a11y][Tabs] Number of Tabs not announced correctly by screen readers 

### ISSUE THREAD

**Issue by emma-boardman** [Shopify/polaris-react#3657]
## Issue summary

When testing with NVDA on Windows (Firefox), and Voiceover on Chrome (Mac), the correct number of tabs in the tab list is not announced. 

JAWs on Windows (Chrome) announces the tabs correctly.

Video: https://streamable.com/ecm5fj

## Expected behavior

When I focus on a tab, I expect the tab to be announced as "1 of 4". 

The expected behaviour can be observed using VoiceOver on Chrome for the [W3 Tab example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html).

## Actual behavior

Each tab is announced as '1 of 1'. 

## Steps to reproduce the problem

Mac:
1. Open https://xqxly.csb.app/ in Chrome
2. Turn on Voiceover
3. Navigate to first tab
4. Right arrow through other tabs 

Windows:
1. Open https://xqxly.csb.app/ in Firefox
2. Turn on NVDA
3. Navigate to first tab
4. Right arrow through other tabs 

## Reduced test case

https://xqxly.csb.app/

## Specifications

- Are you using the React components? (Y/N): Y 
- Polaris version number: in OnlineStore UI: 5.8.4, in reduced test case Sandbox: 5.10.1 
- Browser: Chrome on macOS, Firefox on Windows
- Device: Mac / Windows 
- Operating System: Mac 10.15.7

**Comment by emma-boardman** [comment]
cc @ginabak

**Comment by emma-boardman** [comment]
duplicate of #2442

**Comment by ginabak** [comment]
Closing in favour of #3647 🥳
